lib/facets/aggs-all.js

```diff
@@ -103,6 +103,22 @@ module.exports = function (queryParams) {
             terms: { field: 'materials' }
           }
         }
+      },
+      museum: {
+        filter: filter(queryParams),
+        aggs: {
+          museum_filters: {
+            terms: {size: 25, field: 'locations.name.value'}
+          }
+        }
+      },
+      gallery: {
+        filter: filter(queryParams),
+        aggs: {
+          gallery_filters: {
+            terms: {size: 25, field: 'locations.name.value'}
```

Comment:
Because I'm filtering the results when displaying the facets, if I have less than this, only a few of the galleries display, (and only three of the four museums), getting 25 results back ensures I have about ten galleries to display (the default size is 10)